static/js/starred_messages_ui.js

```diff
@@ -15,7 +15,7 @@ export function confirm_unstar_all_messages() {
         parent: modal_parent,
         html_heading: i18n.t("Unstar all messages"),
         html_body,
-        html_yes_button: i18n.t("Unstar messages"),
+        html_yes_button: i18n.t("Confirm"),
```

Comment:
<img width="429" alt="unstar_all_message" src="https://user-images.githubusercontent.com/59444243/113508505-04899a00-956e-11eb-82c9-b714cfee2219.png">
Do we want to change the text to: "Are you sure you want to unstar all messages? This action cannot be undone."

Reply:
Yes, your proposal sounds good.